pycbc/calibration/recalibrate.py

```diff
@@ -242,3 +251,59 @@ def adjust_strain(self, fs=None, qinv=None, delta_fc=None, kappa_c=1.0,
                                           delta_f=self.strain.delta_f)
 
         return strain_adjusted
+
+    @classmethod
+    def tf_from_file(cls, path, delimiter=" "):
+        """Convert the contents of a file with the columns
+        [freq, real(h), imag(h)] to a numpy.array with columns
+        [freq, real(h)+j*imag(h)].
+
+        Parameters
+        ----------
+        path : string
+        delimiter : {" ", string}
+
+        Return
+        ------
+        numpy.array
+        """
+        data = numpy.loadtxt(path, delimiter=delimiter)
+        freq = data[:, 0]
+        h = data[:, 1] + 1.0j * data[:, 2]
+        return numpy.array([freq, h]).transpose()
+
+    def from_config(self, cp, ifo):
+        """Read a config file to get calibration options and transfer
+        functions which will be used to intialize the model.
+
+        Parameters
+        ----------
+        cp : WorkflowConfigParser
+            An open config file.
+        ifo : string
+            The detector (H1, L1) for which the calibration model will
+            be loaded.
+
+        Return
+        ------
+        list
+        """
+        # read transfer functions
+        tfs = []
+        tf_names = ["a-tst", "a-pu", "c", "d"]
+        for tag in ['-'.join([ifo, "transfer-function", name])
+                    for name in tf_names]:
+            tf_path = cp.get_opt_tag("calibration", tag)
+            tfs.append(self.tf_from_file(tf_path))
+        a_tst0 = tfs[0][:, 1]
+        a_pu0 = tfs[1][:, 1]
+        c0 = tfs[2][:, 1]
+        d0 = tfs[3][:, 1]
+        freq = tfs[0][:, 0]
+
+        # read fc0, fs0, and qinv0
+        fc0 = cp.get_opt_tag("calibration", '-'.join([ifo, "fc0"]))
```

Comment:
Also you had ``section`` in the function above but hardcoded ``"calibration"`` here. Pass ``section`` to this function and use it here.